tardis/simulation/tests/test_simulation.py

```diff
@@ -112,6 +112,9 @@ def test_plasma_estimates(simulation_one_loop, refdata, name):
 def test_plasma_state_iterations(simulation_one_loop, refdata, name):
     actual = getattr(simulation_one_loop, name)
 
+    if hasattr(actual, "value"):
+        actual = actual.value
```

Comment:
This strips units off?

Reply:
Could use a comment about why it is here

Reply:
yes - it gets converted to pandas Series - and that doesn't work with the new pandas

Reply:
This is added but it doesn't show up - no idea why - see latest commit